netplay/waitroom.cpp

```diff
@@ -12,6 +12,7 @@ WaitRoom::WaitRoom(QString filename, QJsonObject room, QWebSocket *socket, QWidg
     player_name = room.value("player_name").toString();
     room_port = room.value("port").toInt();
     room_name = room.value("room_name").toString();
+    input_delay = room.value("input_delay").toInt();
```

Comment:
This value doesn't appear to be used anywhere

Reply:
good catch - removed